dmaws/commands/deploy.py

```diff
@@ -24,3 +27,26 @@ def deploy_cmd(ctx, repository_path):
         sys.exit(1)
 
     ctx.log("URL: http://%s/", url)
+
+
+@click.argument('deployments_json', nargs=1, type=click.File())
+@click.option('--github-token', help="Github API token", default=None)
+@base.command('publish-deployments')
+def publish_releases(deployments_json, github_token):
+    """Publishes Jenkins deployments JSON dump to Github Deployments API"""
+
+    releases = json.load(deployments_json)
+    return all(
+        publish_deployment(
```

Comment:
I think we want those keys to exist, otherwise we're not publishing the correct data. That JSON is manually built at the moment and we shouldn't normally need this once each release publishes it's own data.